src/cljs/rems/application.cljs

```diff
@@ -121,37 +121,33 @@
    (prn errors)
    (assoc-in db [::edit-application :validation-errors] errors)))
 
-(defn- save-application! [description application-id catalogue-ids field-values accepted-licenses]
+(defn- save-application! [description application-id field-values accepted-licenses]
   (status-modal/common-pending-handler! description)
-  (post! "/api/applications/save"
+  (post! "/api/applications/command"
          {:handler (partial status-modal/common-success-handler! #(rf/dispatch [::enter-application-page application-id]))
           :error-handler status-modal/common-error-handler!
-          ;; TODO change the API to match the draft-saved event's structure
-          :params (merge {:command "save"
-                          :items field-values
-                          :licenses (into {} (for [license-id accepted-licenses]
-                                               [license-id "approved"]))}
-                         (if application-id
-                           {:application-id application-id}
-                           {:catalogue-items catalogue-ids}))}))
+          ;; TODO change the command to match the draft-saved event's structure
+          :params {:type :rems.workflow.dynamic/save-draft
```

Comment:
hmm so we no longer write the application_text_values and application_license_approval_values tables?

Reply:
Yes